modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/aquatic.dm

```diff
@@ -32,12 +32,12 @@
 	changesource_flags = MIRROR_BADMIN | WABBAJACK | MIRROR_MAGIC | MIRROR_PRIDE | ERT_SPAWN | RACE_SWAP | SLIME_EXTRACT
 	examine_limb_id = SPECIES_AKULA
 	bodypart_overrides = list(
-		BODY_ZONE_HEAD = /obj/item/bodypart/head/mutant/akula,
-		BODY_ZONE_CHEST = /obj/item/bodypart/chest/mutant/akula,
-		BODY_ZONE_L_ARM = /obj/item/bodypart/arm/left/mutant/akula,
-		BODY_ZONE_R_ARM = /obj/item/bodypart/arm/right/mutant/akula,
-		BODY_ZONE_L_LEG = /obj/item/bodypart/leg/left/mutant/akula,
-		BODY_ZONE_R_LEG = /obj/item/bodypart/leg/right/mutant/akula,
+		BODY_ZONE_HEAD = /obj/item/bodypart/head/mutant,
+		BODY_ZONE_CHEST = /obj/item/bodypart/chest/mutant,
+		BODY_ZONE_L_ARM = /obj/item/bodypart/arm/left/mutant,
+		BODY_ZONE_R_ARM = /obj/item/bodypart/arm/right/mutant,
+		BODY_ZONE_L_LEG = /obj/item/bodypart/leg/left/mutant,
+		BODY_ZONE_R_LEG = /obj/item/bodypart/leg/right/mutant,
```

Comment:
I'd probably turn the `/akula` variants that we used to have into `/aquatic`, so they can still have access to it, and that it's not just lost.

Reply:
No, they're awful body sprites; they are better off gone.

Reply:
While I do get that, I feel like removing those sprites should be done in a separate PR, so people have less reasons to complain that "waa my character doesn't look the same anymore waa". I'm not against it, I just want to reduce the amount of friction.

We can't tell people "go Aquatic, your character will be the exact same as it was before" if they will not, in fact, be the exact same as they were before.